iosApp/iosAppTests/Pages/Onboarding/OnboardingPageTests.swift

```diff
@@ -16,21 +16,24 @@ final class OnboardingPageTests: XCTestCase {
     @MainActor func testFlow() throws {
         let onboardingRepository = MockOnboardingRepository()
         let finishExp = expectation(description: "calls onFinish")
+        let stepChannel = PassthroughSubject<Void, Never>()
+
         let sut = OnboardingPage(
             screens: OnboardingScreen.allCases,
             onFinish: { finishExp.fulfill() },
-            onboardingRepository: onboardingRepository
+            onAdvance: { stepChannel.send() },
+            onboardingRepository: onboardingRepository,
+            // Actual button location dialogue handling is unit tested in OnboardingScreenView.testLocationFlow
+            skipLocationDialogue: true
```

Comment:
I don't love this skip, but it seemed to be the most straightforward way to test.